### DISCUSSION TITLE
Not working? 🔧

### DISCUSSION THREAD

**Discussion by arifszn** [arifszn/gitprofile#548] [General]
This discussion is dedicated to troubleshooting issues you might encounter while deploying **GitProfile**. Feel free to share any problems you're facing, and I will do my best to assist you.

**Comment by Garicore01** [comment]
Hi! I've just deploy the project but when I go to https://garicore01.github.io/ I can't see anything

**Reply by arifszn** [reply]
If you want your portfolio to be available at `https://garicore01.github.io`:
- Rename the project to `garicore01.github.io` in GitHub. Currently, your project name is `portfolio`.
- Go to project's **Settings** -> **Pages** -> **Build and deployment** and double check if the **Source** is `GitHub Actions`.
- Finally, change the `base` property in line 12 to `'/'` in `gitprofile.config.ts`:

    ```ts
      base: '/',
    ```

**Reply by Garicore01** [reply]
I've just done these changes but It still with the blank page

**Reply by arifszn** [reply]
What about this step?
`Enable workflows: Go to your repo's Actions tab and enable workflows.`

**Reply by Garicore01** [reply]
![image](https://github.com/arifszn/gitprofile/assets/125468327/f90ad8f3-e3bd-4b7a-ad89-006e6573ec6a)

**Reply by arifszn** [reply]
Provide a screenshot of Settings -> Pages -> Build and deployment

**Reply by Garicore01** [reply]
![image](https://github.com/arifszn/gitprofile/assets/125468327/66aad7ab-2245-4fda-b1e6-0e3f79276d88)

**Reply by arifszn** [reply]
All looks good. Can you try pushing a commit to trigger the workflow?

**Reply by Garicore01** [reply]
I've deleted the repository and create a new fork. It's works!
Thanks!!

**Reply by Garicore01** [reply]
How I can put the Education set on the top of the page?

**Reply by arifszn** [reply]
You can adjust the orders at `src/components/gitprofile.tsx`.

**Comment by aphotinakis** [comment]
Having an issue with my website displaying a blank page. Not sure why this is happening. 
![Screenshot 2024-06-10 110738](https://github.com/arifszn/gitprofile/assets/156950631/a5ad0ab4-2cba-43bb-beb0-a88e22c3b0be)

**Reply by arifszn** [reply]
Can you share your repository URL? If not possible, share a screenshot of the repository's **Actions** tab.

**Reply by aphotinakis** [reply]
https://github.com/acphotinakis/acphotinakis.github.io

And here is a SS: 
![Screenshot 2024-06-10 113558](https://github.com/arifszn/gitprofile/assets/156950631/f198c96b-7597-4df4-aa45-ab9d431bf9ea)
![Screenshot 2024-06-10 113612](https://github.com/arifszn/gitprofile/assets/156950631/8caea8a0-3e33-424c-a2fb-edf3a5bc5397)

**Reply by arifszn** [reply]
From Repository Settings -> Pages -> Build and deployment, double check if the Source is GitHub Actions.

The GitHub Actions are not getting triggered. Another solution is to delete the repository and fork it again.

**Reply by aphotinakis** [reply]
It works now. I think it just needed time to reflect the changes. Thank you for your help!

**Reply by ZaidlKhan** [reply]
I am having the same problem here is my repo link:
https://github.com/ZaidlKhan/zk_portfolio

**Comment by anthony1204** [comment]
i cant get it to deploy to my webpage, https://anth.lol
https://github.com/anthony1204/gitprofile/tree/main here is the repository

i will give you access to the repository if i can

**Reply by anthony1204** [reply]
its pending
<img width="817" alt="image" src="https://github.com/user-attachments/assets/c7eb48ac-4714-4848-be72-f2db61173e00">

**Reply by arifszn** [reply]
Sorry, I was away from internet. Are you still facing the issue? I can see your website is live now at https://anthony1204.github.io/gitprofile/.

**Comment by Charanrajueluri123** [comment]
could help me i am getting white page
![charan](https://github.com/user-attachments/assets/ca8926be-fea4-4e88-b9df-ee8499487341)

**Reply by Charanrajueluri123** [reply]
this is settings page 
![image](https://github.com/user-attachments/assets/3f8b36cd-23e7-4b11-8445-2725b827aa2f)

**Reply by arifszn** [reply]
Change the base value in `gitprofile.config.ts`.

```
base: '/charanraju/'
```

**Comment by Knight7561** [comment]
I did try Many of these comments when I downloaded the repo and make changes and took it to my git. Basically the issue was the workflows folder under `.github/workflow/deploy.yaml` wasn't downloaded, so once I re created the file same as OP's file. Everything worked out. 
Thanks @arifszn for the git. But I wish this detail was in the readme file as well.

**Comment by dev4pk** [comment]
It didn't work on mine, I forked it and changed the name but the page appeared blank

![image](https://github.com/user-attachments/assets/b1fe97ac-648f-44d1-a838-c993b892fc3a)

**Comment by anthony1204** [comment]
tried to add it to my domain but didn’t work so i wrote my own

On 27 Oct 2024, at 6:50 AM, dev4pk ***@***.***> wrote:



It didn't work on mine, I forked it and changed the name but the page appeared blank

image.png (view on web)<https://github.com/user-attachments/assets/b1fe97ac-648f-44d1-a838-c993b892fc3a>

**Reply by arifszn** [reply]
Give me the repository link.

**Reply by dev4pk** [reply]
Repositorie: https://github.com/dev4pk/dev4pk.github.io

Site: https://dev4pk.github.io/

**Reply by arifszn** [reply]
Change the base value in `gitprofile.config.ts`:

```
base: '/'
```

**Reply by dev4pk** [reply]
Thanks

**Comment by augermora** [comment]
The Articles section (blog) is not working properly.

I've created an article in DEV.TO, changed the gitprofile blog properties source to 'dev' and username to 'augermora' but it's now showing up any post.

![image](https://github.com/user-attachments/assets/5c7df565-3033-4886-ae30-e53a7d7d045f)

Link to my repository:
[https://github.com/augermora/augermora.github.io/](https://github.com/augermora/augermora.github.io/)

Link to my DEV profile: 
[https://dev.to/augermora](https://dev.to/augermora)

Any help would be really appreciated. Also, good job with this Github Page, it looks amazing.

**Reply by arifszn** [reply]
> The Articles section (blog) is not working properly.
> 
> I've created an article in DEV.TO, changed the gitprofile blog properties source to 'dev' and username to 'augermora' but it's now showing up any post.
> 
> ![image](https://private-user-images.githubusercontent.com/45043388/419149145-5c7df565-3033-4886-ae30-e53a7d7d045f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDExMTU1OTEsIm5iZiI6MTc0MTExNTI5MSwicGF0aCI6Ii80NTA0MzM4OC80MTkxNDkxNDUtNWM3ZGY1NjUtMzAzMy00ODg2LWFlMzAtZTUzYTdkN2QwNDVmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzA0VDE5MDgxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU0NmY2NWVmYjM2MmEyNDIwOTVlNTg4MDVlZGQ4ZmY0ZDVmNzYzMTI0NzI1YTdmOGU3NDQyZDlmZmY4Nzg3ZjAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.wiJH-qRv-WXomQmycKH5-wJNhv5S8VgcfQpAyGKvAhQ)
> 
> Link to my repository: https://github.com/augermora/augermora.github.io/
> 
> Link to my DEV profile: https://dev.to/augermora
> 
> Any help would be really appreciated. Also, good job with this Github Page, it looks amazing.

Dev.to's API caches the articles, and it usually takes some time to invalidate the cache. For more info, you can check this https://github.com/arifszn/gitprofile/issues/69.

**Reply by augermora** [reply]
Thanks for the quick reply!

I didn't know about DEV API Cache, I just checked it out and it works fine. My bad.

Keep up the good work 🙂

**Comment by Cartev112** [comment]
My link shows only a white page with 
<%- googleAnalyticsScript %>

**Reply by arifszn** [reply]
Your workflow is not running properly https://github.com/Cartev112/Cartev112.github.io/actions.

**Comment by dapamd** [comment]
Deploy with netlify, all done but ......
Whats wrong?

**Reply by dapamd** [reply]
Only shows 
<%- googleAnalyticsScript %>

**Reply by dapamd** [reply]
Here's my link https://akdpa.my.id

**Reply by arifszn** [reply]
Refer to this link https://vite.dev/guide/static-deploy#netlify-with-git.

Make sure you are pointing the `dist` folder for deploying.
